LayoutTests/editing/execCommand/remove-format-mixed-editable-crash.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script>
+if (window.testRunner)
+    testRunner.dumpAsText();
+
+onload = function() {
+    document.execCommand('SelectAll');
+    document.execCommand('RemoveFormat');
+    document.body.textContent = 'PASS if Webkit doesn\'t crash.';
```

Comment:
I don't think it's accurate to say doesn't "crash" since we're talking about debug assertion here.
I suggest we rephrase it as PASS if WebKit doesn't hit any debug assertions.

Reply:
Updated to just mention "PASS". Thanks!